openapi/components/schemas/OrganizationId.yaml

```
@@ -0,0 +1,5 @@
+type: string
+readOnly: true
+description: Organization ID.
```

nitpick: Good to add more detail here
```suggestion
description: |-
     Your organization ID. The organization ID describes your
     organization in Rebilly. For information on how to obtain
     your organization ID, see
     [Obtain your organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
```